src/Microsoft.Framework.Runtime.Hosting/Json/JsonValue.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Runtime.Json
+{
+    internal class JsonValue
+    {
+        public JsonValue(JsonPosition position)
```

Comment:
I think `JsonToken` and `JsonValue` are swapped because `JsonToken` has a value while `JsonValue` does not